sonar-project.properties

```diff
@@ -1,7 +1,7 @@
 sonar.projectKey=starlite-api_starlite
 sonar.organization=starlite-api
 sonar.coverage.exclusions=**/__init__.py, **/*.js
-sonar.cpd.exclusions=starlite/params.py, starlite/handlers/*, starlite/connection.py
+sonar.cpd.exclusions=starlite/params.py, starlite/handlers/*, starlite/connection.py, starlite/contrib/jwt/jwt_auth.py
```

Comment:
Why? (Just curious)

Reply:
because the login method is duplicated for parameters but has a different body